src/services/consortiaService.js

```diff
@@ -32,6 +34,20 @@ export const fetchConsortium = ({ okapi }, tenant) => {
     .then(data => data.consortia[0]);
 };
 
+export const fetchConsortiumMembers = async ({ okapi, user }) => {
+  const centralTenant = user?.user?.consortium?.centralTenantId || await fetchConsortiaCentralTenant({ okapi });
```

Comment:
I think you can use just okapi.tenant as login is always done from central tenant

Reply:
But I'm wondering if this service will be used  not only for login, for instance, `useConsortiumMembers` use that to fetch tenants, and it might be used from non-central tenant

Reply:
Nevermind, I've updated